impeller/typographer/lazy_glyph_atlas.cc

```diff
@@ -50,8 +51,8 @@ std::shared_ptr<GlyphAtlas> LazyGlyphAtlas::CreateOrGetGlyphAtlas(
     i++;
     return &result;
   };
-  auto atlas =
-      text_context->CreateGlyphAtlas(type, std::move(atlas_context), iterator);
+  auto atlas = text_context->CreateGlyphAtlas(type, std::move(atlas_context),
+                                              iterator, scale);
```

Comment:
The scale needs to be updated every time the transformation matrix changes (there's an earlier cache return in this method), otherwise text will render at the wrong size.

Reply:
Er, like @jonahwilliams said, this would actually need to be done for all of the frames here. But I don't think there's enough context to connect those frames to their respective `TextContents` here.